models/inventory-entry/src/types.ts

```diff
@@ -7,6 +7,9 @@ import type { TReferenceGraphql } from '@commercetools-test-data/commons';
 import type { TBuilder } from '@commercetools-test-data/core';
 
 // Default
+/**
+ * @deprecated use `TInventoryEntryRest` instead
+ */
 export type TInventoryEntry = Omit<InventoryEntry, 'supplyChannel'> & {
   supplyChannel: Channel;
 };
```

Comment:
For this PR do I need to do anything about the `InventoryEntryDraft`? I didn't see anything about that in the docs.

Reply:
Yes, we need to migrate that model as well.